Firebase/Database/Core/FRepoManager.m

```diff
@@ -76,43 +79,43 @@ + (FRepo *) createRepo:(FRepoInfo *)repoInfo config:(FIRDatabaseConfig *)config
 
 + (void) interrupt:(FIRDatabaseConfig *)config {
     dispatch_async([FIRDatabaseQuery sharedQueue], ^{
-        NSMutableDictionary *configs = [FRepoManager configs];
-        NSMutableDictionary *repos = configs[config.sessionIdentifier];
-        for (FRepo* repo in [repos allValues]) {
-            [repo interrupt];
-        }
+      FRepoDictionary *configs = [FRepoManager configs];
+      NSMutableDictionary<FRepoInfo *, FRepo *> *repos = configs[config.sessionIdentifier];
+      for (FRepo *repo in [repos allValues]) {
+          [repo interrupt];
```

Comment:
Nit: Spacing here is inconsistent with rest of the file.

Reply:
I was hoping I could get the clang-format PR to build before submitting this, but it looks like that is not yet happening... I fixed the formatting here.